infra/launch_on_ray.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/python
+# Similar to launch.py, but this instead launches on a Ray cluster configured with auto-scaling TPUs
+
+import argparse
+import getpass
+import os
+import tempfile
+import time
+from pathlib import Path
+
+import draccus
+from ray.dashboard.modules.job.common import JobStatus
+from ray.dashboard.modules.job.sdk import JobSubmissionClient
+
+import levanter.infra.cli_helpers as cli
+import levanter.infra.docker as docker
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    config = cli.load_config()
+
+    cli.add_arg(parser, config, ["--docker_base_image"], default="ghcr.io/stanford-crfm/levanter-base:latest")
```

Comment:
Why isn't this all just draccus specified?

Reply:
Well, Draccus/Pyrallis are designed for fairly simple CLIs and can't support some of the fluent docker-like stuff you'd want for this. In particular, handling repeated lists for env vars (like docker run) and support for unused args. I could add those things to Draccus, but it incurs a complexity cost.

Reply:
It seems like supporting lists in draccus would be pretty useful in general.

Reply:
it supports lists, just not `-e WANDB_MODE offline -e HF_KEY admajkldmakld`